dive_sailthru_client/client.py

```diff
@@ -24,7 +29,40 @@ class DiveEmailTypes:
     Spotlight = "spotlight"
 
 
-class DiveSailthruClient(SailthruClient):
+# There is some skullduggery below in order to override the hardcoded 10 second timeout on HTTP requests
+# per TECH-3849. First we copy/paste the sailthru_http_request() function that originally exists here:
+# https://github.com/sailthru/sailthru-python-client/blob/521fdaa30890a29da8fbb02726e7d22ed174b878/sailthru/sailthru_http.py#L30
```

Comment:
this referenced version allowed for a `headers` parameter as well though the below copy does not include it, did you take it out here on purpose?

Reply:
Honestly just wrote my version before looking at theirs. I guess it's better to expose the things that anyone might want to change as parameters, but this is an internal function anyway so it's not like the end user could use `headers` -- it'd be for use in subclasses of the API Client

Reply:
I was worried that some higher yet still internal level method in sailthru client accepts `headers` and depends on being able to pass them down to this method eventually (in other words, worried about getting getting a TypeError a la "takes exactly X arguments, X+1 given" if we called something that tried to send `headers` all the way down). I did a browse and it doesn't seem so; only for a few high level API cases for the end user to supply (i.e. https://github.com/sailthru/sailthru-python-client/blob/17e201ccfbae747ce01b71a7fd1f8470cd51621c/sailthru/sailthru_client.py#L516) which we don't even use these higher level methods in our client/possibly anywhere ever.